docs/plugins/backup.md

```diff
@@ -0,0 +1,56 @@
+# Moonshot backup plugin
+
+Moonshot plugin for backing up config files.
+
+## Functionality
+
+The plugin collects and deflates certain files to a single tarball,
+and uploads that to a a given S3 bucket. The whole process happens
+in memory, nothing is written to disk. The plugin currently supports single files only, 
+including whole directories in your tarball is not possible yet.
+
+The plugin uses the Moonshot AWS config, meaning that the bucket must be
```

Comment:
A bit of a nitpick, but this isn't quite true. It means that whatever credentials you are using locally must have access to the bucket. Cross-account permissions are a possibility.